src/sentry/api/endpoints/organization_stats_summary.py

```diff
@@ -0,0 +1,258 @@
+            "the `sum(quantity)` field is bytes for attachments, and all others the 'event' count for those types of events.\n\n"
+            "`sum(times_seen)` sums the number of times an event has been seen. "
+            "For 'normal' event types, this will be equal to `sum(quantity)` for now. "
+            "For sessions, quantity will sum the total number of events seen in a session, while `times_seen` will be the unique number of sessions. "
+            "and for attachments, `times_seen` will be the total number of attachments, while quantity will be the total sum of attachment bytes."
+        ),
+    )
+
+    # filter parameters
+
+    project = serializers.ListField(
+        required=False,
+        help_text="The ID of the projects to filter by.",
+    )
+
+    category = serializers.ChoiceField(
+        ("error", "transaction", "attachment", "replays", "profiles"),
+        required=False,
+        help_text=(
+            "If filtering by attachments, you cannot filter by any other category due to quantity values becoming nonsensical (combining bytes and event counts).\n\n"
+            "If filtering by `error`, it will automatically add `default` and `security` as we currently roll those two categories into `error` for displaying."
+        ),
+    )
+    outcome = serializers.ChoiceField(
+        [o.name.lower() for o in Outcome],
+        required=False,
+        help_text="See https://docs.sentry.io/product/stats/ for more information on outcome statuses.",
+    )
+
+    reason = serializers.CharField(
+        required=False, help_text="The reason field will contain why an event was filtered/dropped."
+    )
+
+    # download the file
+    download = serializers.BooleanField(
+        help_text=("Download the API response in as a csv file"),
+        required=False,
+    )
+
+
+class _ProjectSummaryStats(TypedDict):  # this response is pretty dynamic, leaving generic
+    id: int
+    slug: str
+    stats: List[Dict[str, Any]]
+
+
+class StatsSummaryApiResponse(TypedDict):
+    start: str
+    end: str
+    projects: List[_ProjectSummaryStats]
+
+
+@extend_schema(tags=["Organizations"])
+@region_silo_endpoint
+class OrganizationStatsSummaryEndpoint(OrganizationEventsEndpointBase):
+    publish_status = {"GET": ApiPublishStatus.PUBLIC}
+    owner = ApiOwner.ENTERPRISE
+
+    @extend_schema(
+        operation_id="Retrieve Summarized Event Counts for Projects in an Organization",
+        parameters=[GlobalParams.ORG_SLUG, OrgStatsSummaryQueryParamsSerializer],
+        request=None,
+        responses={
+            200: inline_sentry_response_serializer("OutcomesResponse", StatsSummaryApiResponse),
+            401: RESPONSE_UNAUTHORIZED,
+            404: RESPONSE_NOT_FOUND,
+        },
+        examples=OrganizationExamples.RETRIEVE_SUMMARY_EVENT_COUNT,
+    )
+    def get(self, request: Request, organization) -> HttpResponse:
+        """
+        Query summarized event counts by project for your Organization.
```

Comment:
Would be nice to have a reference here to the other API and say `if you want to do X see that other one`. https://docs.sentry.io/api/organizations/retrieve-event-counts-for-an-organization-v2/